File: CODES/2) PYTHON/4) IF/5.py
# a = int(input("a = "))
# b = int(input("b = "))
# c = int(input("c = "))
# d = 0
# e = 0
# if a > 0:
#     d = d + 1
# elif a < 0:
#     e = e + 1
# if b > 0:
#     d = d + 1
# elif b < 0:
#     e = e + 1
# if c > 0:
#     d = d + 1
# elif c < 0:
#     e = e + 1
# print(d, "ta musbat")
# print(e, "ta manfiy")

a = int(input("a = "))
b = int(input("b = "))
c = int(input("c = "))
print(sum(x > 0 for x in [a, b, c]), "ta musbat")
print(sum(x < 0 for x in [a, b, c]), "ta manfiy")

File: LANGUAGES/4) Php/4) IF/5.php
<?php
$a = 5;
$b = 7;
$c = -2;
$MusbatCount = 0;
$ManfiyCount = 0;

if ($a > 0) $MusbatCount++;
else if ($a < 0) $ManfiyCount++;
if ($b > 0) $MusbatCount++;
else if ($b < 0) $ManfiyCount++;
if ($c > 0) $MusbatCount++;
else if ($c < 0) $ManfiyCount++;

echo $MusbatCount . " Ta musbat <br>";
echo $ManfiyCount . " Ta Manfiy ";
